### DISCUSSION TITLE
Horizontal-component cross-correlations with sensors not being aligned to north

### DISCUSSION THREAD

**Discussion by fablindner** [ROBelgium/MSNoise#265] [Q&A] [answered]
Hi! I have several stations that are not aligned to north (channels are Z,2,3), which all have different orientations, but the azimuths were measured and stored in the station XML files. What's the best practice to calculate horizontal-component cross-correlations from these stations? Is there a way around to manually rotate the components to Z,N,E, save the seismograms to disk and then scan the new files with MSNoise? The question is basically, if MSNoise can handle components other than Z,N,E.

**Comment by ThomasLecocq** [comment, marked as answer]
Hey Fabian,
Naaah, not easily sadly, I never added this func to automatically rotate the components based on resp (simply because I didn't have azimuth-info in my response files before).
About working with other components, using the latest dev (master) version, it should work ok, e.g.:
``components_to_compute=23,2Z,3Z``

and all pairs will be done.
